_source/_posts/2023-01-24-jakartaee-auth0.md

```diff
@@ -0,0 +1,545 @@
+---
+layout: blog_post
+title: "Open ID authentication with Jakarta EE 10 and Security 3.0"
+author: andrew-hughes
+by: contractor
+communities: [java,security]
+description: "Use Jakarta EE 10 to build a secure Java web application using Open ID connect and Auth0."
+tags: [java, jakartaee]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+## Open ID authentication with Jakarta EE 10 and Security 3.0
+
+Jakarta EE 10 includes a new authentication mechanism: Open ID Connect! This can be added to a Jakarta EE servlet using the new `@OpenIdAuthenticationMechanismDefinition` annotation. 
```

Comment:
```suggestion
Jakarta EE 10 includes a new authentication mechanism: OpenID Connect! This can be added to a Jakarta EE servlet using the new `@OpenIdAuthenticationMechanismDefinition` annotation. 
```

Reply:
@moksamedia Please change `Open ID` to `OpenID` throughout. https://auth0.com/docs/authenticate/protocols/openid-connect-protocol